demo-vue/src/components/VueState.vue

```diff
@@ -0,0 +1,135 @@
+        <span v-for="gender in genders" :key="gender">
+          <input :id="'gender-' + gender" name="gender" type="radio"
+                 :value="gender" v-model="form.gender">
+          <label :for="'gender-' + gender">{{ gender }}</label>
+        </span>
+        <br>
+        <label>{{ genderState.validationMessage }}</label>
+      </div>
+      <div class="formfield">
+        <input id="username" name="username" type="text" aria-placeholder="Username"
+               :class="usernameState.validationClassName" v-model.trim="form.username">
+        <label for="username"
+               :data-validation-message="usernameState.validationMessage">Username</label>
+      </div>
+      <div class="formfield">
+        <input id="password" name="password" type="password" aria-placeholder="Password"
+               :class="passwordState.validationClassName" v-model="form.password">
+        <label for="password"
+               :data-validation-message="passwordState.validationMessage">Password</label>
+      </div>
+      <div class="formfield">
+        <input id="confirmPassword" name="confirmPassword" type="password" aria-placeholder="Confirm password"
+               :class="confirmPasswordState.validationClassName" v-model="form.confirmPassword">
+        <label for="confirmPassword"
+               :data-validation-message="confirmPasswordState.validationMessage">Confirm password</label>
+      </div>
+      <div class="formfield-checkbox" :class="acceptsTermsState.validationClassName">
+        <div>
+          <input id="acceptsTerms" name="acceptsTerms" type="checkbox" v-model="form.acceptsTerms">
+          <label for="acceptsTerms">
+            Accept <a href="#">terms and conditions</a>
+          </label>
+        </div>
+        <label>{{ acceptsTermsState.validationMessage }}</label>
+      </div>
+      <div class="formactions">
+        <div>
+          <button @click="stricklandOnSubmit">Submit</button>
+        </div>
+        <div>
+          {{ isValid ? 'Can Submit' : 'Cannot Submit Yet' }}
+        </div>
+      </div>
+    </div>
+    <pre id="current-state">{{ JSON.stringify({ form, validation, validationHistory }, null, 2) }}</pre>
+  </div>
+</template>
+
+<script>
+import formValidator from '../validators/formValidator.js';
+import stricklandMixinFactory from '../mixins/vueStricklandMixinFactory.js';
+import mapFormFieldValidationStates from '../mappers/vueStricklandMappers.js';
+
+let form = {
+  firstName: null,
+  lastName: null,
+  age: null,
+  gender: null,
+  username: null,
+  password: null,
+  confirmPassword: null,
+  acceptsTerms: false
+};
+
+export default {
+  name: 'VueState',
+  mixins: [
+    stricklandMixinFactory(formValidator)
+  ],
+  data: () => ({
+    form,
+    validationDependencies: {
+      password: ['confirmPassword']
+    },
+    // TODO: Do we want this feature?
+    // Idea is if you define this then you get some debug help
+    validationHistory: [],
```

Comment:
I suggest removing it so that it doesn't look like a concept you _have to_ adopt.

Reply:
Just to be clear, do you mean remove it from the example (so people don't feel they need to adopt it) or remove the logging entirely? 

It's been helping me a bit lately so I''ll keep around until the PR is ready, but will likely get rid of it entirely.